source/_components/sensor.pjm.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: "PJM"
+description: "Instructions on how to set up the PJM  sensor in Home Assistant."
+date: 2018-06-14
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: pjm.png
+ha_category: Energy
+ha_release: "0.72"
```

Comment:
Home Asssistant `0.72` has already been shipped. Please update this to next major release, currently `0.73`.